binding/format/protobuf/v2/pb/cloudevent.pb.go

```diff
@@ -7,7 +7,8 @@
 package pb
 
 import (
-	any "github.com/golang/protobuf/ptypes/any"
+	any1 "github.com/golang/protobuf/ptypes/any"
```

Comment:
Why replace `any` with `any1`? Is this intentional？

Reply:
This code is generated by proto compiler, not manually altered.